### DISCUSSION TITLE
Permanent files using <Files> in Wix v5

### DISCUSSION THREAD

**Discussion by AntonyNifosi** [wixtoolset/issues#8833] [Questions]
I am using Wix V5 to create a MSI installer.
I use the `<Files>` to harvest files, and I would like to have them permanent, meaning it's not removed at uninstall.
I only see the possibility to do it on a `<Component>` but can't use Files in a Component.
So how I could achieve this ?

I would like to do something like this :
```
 <Files Include="A/B/**" Permanent="yes" />
```

I already check the documentation but didn't notice any argument I could use to set the files harvested as permanent one.

**Comment by chrpai** [comment]
I don't think this is a supported use case for Files.  Are you using a bootstrapper?  You could put all these files in a micropackage and make the entire package permanent.
